apps/crn-server/test/routes/team.route.test.ts

```diff
@@ -150,50 +150,7 @@ describe('/teams/ route', () => {
 
       await supertest(app).get('/teams').query(params);
 
-      const expectedParams: FetchTeamsOptions = {
-        ...(params as Required<FetchOptions>),
-        showTeamTools: [loggedUser.teams[0]!.id],
-      };
-
-      expect(teamControllerMock.fetch).toBeCalledWith(expectedParams);
-    });
-
-    describe('Team tools', () => {
-      test('Should select the team tools for the teams the user is a member of', async () => {
```

Comment:
we are not showing team tools for each team now, right, the tools are just no longer part of the list response?